arm-asazure/2016-05-06/swagger/asazure.json

```diff
@@ -418,6 +430,7 @@
       "in": "path",
       "required": true,
       "type": "string",
+      "pattern": "^[-\\w\\._]+$",
```

Comment:
It would be nice to define commonly used model definitions and parameters in a common file and ask everyone to reference from there. That makes things consistent.